src/orca-loader/Hyper.res

```diff
@@ -331,10 +331,14 @@ let make = (publishableKey, options: option<JSON.t>, analyticsInfo: option<JSON.
                 }
                 postSubmitMessage(dict)
 
-                if (
-                  isSdkButton ||
-                  (val->JSON.Decode.bool->Option.getOr(false) && redirect === "always")
-                ) {
+                if isSdkButton {
+                  if !(val->JSON.Decode.bool->Option.getOr(false)) {
+                    resolve1(json)
+                  } else {
+                    resolve1(data)
```

Comment:
remove resolve in 2 case as I guess it's not needed. @ArushKapoorJuspay

Reply:
removed resolve1(data)